docs/src/documentation/03-components/02-structure/table/03-accessibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/table/accessibility/
+---
+
+# Accessibility
```

Comment:
In some other sections we're using h2 here, as we have an h1 in the box above.
No strong opinion, as we don't have this done consistently. Feel free to leave it like it is.

Reply:
Yeah, last time Daniel pointed it out when I changed it to h2 😅 https://github.com/kiwicom/orbit/pull/4622#discussion_r1955885165